### PR TITLE
Editor: change UI state to reflect the result of pre-population

### PR DESCRIPTION
**Pull request by jozzi**
Uploading operator manifest marks all sections for review so user is visually notified that something changed

All the editor sections are marked for review as we cannot simply determine which section have changed.
In order to achieve this it would require major refactor of the entire editor functionality as editor page does not know which part of operator maps to which section and how to validate it / determine if its pristine.

### CODE REVIEW

**Comment by jeff-phillips-18** [comment]
We could create more constants like [METADATA_FIELDS](https://github.com/operator-framework/operatorhub.io/blob/editor/frontend/src/pages/OperatorEditorPage/OperatorMetadataPage.js#L17) for each section and use that to compare the operator before and after to know if that section was affected by the load. Thoughts?

**Comment by jozzi** [comment]
Yep, I agree Jeff, we can extract relevant fields for each section out and compare actual operator model against default to see if it has changed.
Just questionable if its worth for such a feature as would touch a lot components, but thats for Daniel to decide :-)

**Comment by jeff-phillips-18** [comment]
not really a lot of components, just add a few constants specifying the fields per section. The a copy of those fields from the before and after operator objects for comparison.

I can write up a function demonstrating what I mean if you'd like.

**Comment by jozzi** [comment]
Not needed, idea is clear to me ;-)

**Comment by jeff-phillips-18** [comment]
@jozzi You will just need to do: 
```
git fetch upstream editor
git rebase upstream/editor
```

**Comment by galletti94** [comment]
@jozzi can you make this PR against the dev branch instead of master?

**Comment by jozzi** [comment]
@galletti94 this branch has upstream branch editor, so no need to change it

**Comment by jozzi** [comment]
Pushed changes so section - fields mapping is in single file and sections are checked as discussed before.

@jeff-phillips-18 ready to be reviewed

File: frontend/src/pages/OperatorEditorPage/editorPageUtils.js [hunk 1]
```diff
@@ -17,6 +17,41 @@ const EDITOR_STATUS = {
   errors: 'errors'
 };
 
+// metadata: 'empty',
+// 'owned-crds': 'empty',
+// 'required-crds': 'empty',
+// deployments: 'empty',
+// permissions: 'empty',
+// 'cluster-permissions': 'empty',
+// 'install-modes': 'empty'
```
[the remark this answers is not in the document]
Comment (jozzi) [reply frontend/src/pages/OperatorEditorPage/editorPageUtils.js]: Oh, yep. It was only helping to keep the section names consistent

**Comment by jozzi** [comment]
From talking to Daniel, this is how it should be, but let's confirm on sync. As nothing changed to previous state (default) section does not need review.

Same applies while creating new operator, you can ignore section if you do not plan to change it, can't you?
@dmesser what do yo think?

**Comment by jeff-phillips-18** [comment]
Agreed to do the recognintion of uploaded values equal to the defaults as a follow-on:
https://trello.com/c/XCq1cUEQ/194-editor-recognize-uploaded-fields-that-are-the-same-as-defaults-and-mark-section-for-review

**Pull request merged**